spec/models/ems_refresh/save_inventory_spec.rb

```diff
@@ -128,24 +128,15 @@
         EmsRefresh.save_vms_inventory(@ems, data)
 
         vms = Vm.all
-        expect(vms.length).to eq(3)
```

Comment:
This spec is now actually testing a VM with the same ems_ref and a new uid_ems get updated and not archived:

```
   128:         data = raw_data_with_dups(@vm1, @vm2)
=> 129:         EmsRefresh.save_vms_inventory(@ems, data)
   130: 
   131:         vms = Vm.all
   132:         expect(vms.length).to eq(2)
   133: 
(byebug) Vm.all.map { |vm| [vm.ems_ref, vm.uid_ems] }
[["vm-0000000000001", "fa7c21a7-2596-45b1-86b6-3ccb8b8c9273"], ["vm-0000000000002", "57a19467-b438-4e44-8710-fe4873066249"]]
(byebug) data.map { |vm| [vm[:ems_ref], vm[:uid_ems]] }
[["vm-0000000000001", "fa7c21a7-2596-45b1-86b6-3ccb8b8c9273"], ["vm-0000000000002", "fa7c21a7-2596-45b1-86b6-3ccb8b8c9273"]]
```

Since the second vm in the data payload has the same ems_ref as an active vm but with a different uid_ems.

Reply:
Do the `it` descriptions reflect these changes (here and the next 2)?  (hard to tell in the diff)

Reply:
The descriptions were pretty general, "handle dups in the raw data" doesn't really describe what it is expecting to happen.

I'll update them to better describe the expected outcome.

Reply:
Done